src/views.lisp

```diff
@@ -178,7 +184,7 @@
 
 (defun render-preferences-page ()
   (let* ((user-prefs (parse-user-preferences))
-         (render-prefs (map 'list
+         (render-prefs (mapcar
                             #'(lambda (pref)
                                 (let* ((keyword-pref (car pref)))
                                   (car (acons keyword-pref (acons :current
```

Comment:
Side-note: When you have a procedure that goes this far to the right, it's generally a sign that you can use some abstraction.
Same for procedures which are too long and condensed, it's generally a sign that too many ideas are being put under 1 name.